chromadb/test/property/strategies.py

```diff
@@ -458,17 +458,26 @@ def recordsets(
     num_unique_metadata: Optional[int] = None,
     min_metadata_size: int = 0,
     max_metadata_size: Optional[int] = None,
+    # ids can only be optional for add operations
+    for_add: bool = False,
```

Comment:
i think this parameter name is confusing and underspecified - can we clarify it or somehow make this less option-heavy?